lib/src/Base/Stat/HMatrixParameters.cxx

```diff
@@ -129,8 +132,10 @@ UnsignedInteger HMatrixParameters::getCompressionMethodAsUnsignedInteger() const
     return 2;
   else if (compressionMethod_ == "ACA+")
     return 3;
+  else if (compressionMethod_ == "ACA random")
```

Comment:
you need to mention in the doc, see HMatrixParameters_doc.i.in

Reply:
Added. Thank you.